src/Aspire.Hosting.Kafka/KafkaContainerImageTags.cs

```diff
@@ -8,4 +8,6 @@ internal static class KafkaContainerImageTags
     public const string Registry = "docker.io";
     public const string Image = "confluentinc/confluent-local";
     public const string Tag = "7.6.0";
+    public const string KafkaUiImage = "provectuslabs/kafka-ui";
```

Comment:
Is this the defacto standard?

Reply:
This the UI suggested by this tutorial https://learn.microsoft.com/en-us/azure/container-apps/tutorial-dev-services-kafka?tabs=bash